tagstudio/src/qt/modals/fix_unlinked.py

```diff
@@ -0,0 +1,179 @@
+# Copyright (C) 2024 Travis Abendshien (CyanVoxel).
+# Licensed under the GPL-3.0 License.
+# Created for TagStudio: https://github.com/CyanVoxel/TagStudio
+
+
+import logging
+import typing
+
+from PySide6.QtCore import QThread, Qt, QThreadPool
+from PySide6.QtWidgets import QWidget, QVBoxLayout, QHBoxLayout, QLabel, QPushButton
+
+from src.core.library import Library
+from src.qt.helpers import FunctionIterator, CustomRunnable
+from src.qt.modals import DeleteUnlinkedEntriesModal, RelinkUnlinkedEntries
+from src.qt.widgets	import ProgressWidget
+
+# Only import for type checking/autocompletion, will not be imported at runtime.
+if typing.TYPE_CHECKING:
+	from src.qt.ts_qt import QtDriver
```

Comment:
This type of importing appears to be a way around the way around the apparent coupling between the Widget and the QtDriver as the widget doesn't need to know anything about the driver, only the IDE for type checking and auto completion